README.md

```diff
@@ -83,20 +100,60 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get("/objects", function(req,res) {
+  res.render("index-page");
+})
+
+app.get("/objects/:name", function(req, res) {
+  Object.findOne({name: req.params.name}).then(function(object) {
+    res.render("objects-show", {
+      objects: objects
+    });
+  });
+});
+
+app.post("/objects", function(req, res) {
+  Object.create(req.body.candidate).then(function(object) {
+    res.redirect("objects/" + Object.name);
+  });
+});
+
+app.post("/objects/:name", function(req, res) {
+  Object.fineOneAndUpdate({name: req.params.name}, req.body.object, function() {
+    res.redirect("/objects/" + Object.name)
+  })
+})
+
```

Comment:
:+1: This has been the case when dealing with HTML forms in a MEN application. But Express does have `.put` and `.delete` methods.